java/client/src/org/openqa/selenium/AppdynamicsCapabilities.java

```diff
@@ -0,0 +1,97 @@
+package org.openqa.selenium;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+/**
+ * Created by shashwat.srivastava on 5/30/17.
+ */
+public class AppdynamicsCapabilities {
```

Comment:
Let's call it `AppdynamicsCapability` since we decided to go for a single capability